trunner/runners/STM32L4Runner.py

```diff
@@ -0,0 +1,87 @@
+#
+# Phoenix-RTOS test runner
+#
+# Stm32l4 runner
+#
+# Copyright 2021 Phoenix SYstems
+# Authors: Mateusz Niewiadomski
+#
+
+import serial
+import time
+import pexpect.fdpexpect
+
+import sys
+from os import path
+from subprocess import Popen, PIPE
+
+from .common import DeviceRunner, Color
+from .common import _BOOT_DIR
+
+
+class STM32L4Runner(DeviceRunner):
+    """ This class provides interface to run test case on STM32L4 target using RaspberryPi.
+        The ST-Link programming board should be connected to USB port
+        and a target should be connected to ST-Link and powered"""
+
+    class oneByOne_fdspawn(pexpect.fdpexpect.fdspawn):
+        """ Workaround for Phoenix-RTOS on stm32l4 targets not processing characters fast enough
+            This class inherits and is passed to harness instead of pexpect.fdpexpect.fdspawn
+            It redefines send() with addition of timeout after each letter sent. Timeout set to 30ms per character"""
+
+        def send(self, s):
+            r = 0
+            for c in s:
+                r += super().send(c)
+                time.sleep(0.05)
+            return r
```

Comment:
I suggest more readable names for variables here

Reply:
Changed variable names to more descriptive